src/models/__tests__/appmodel.test.ts

```diff
@@ -0,0 +1,25 @@
+import { AppModel } from "../appmodel";
+
+const appModel = new AppModel();
+
+describe("text transformation suite", () => {
+
+	it(`should make the transformation text lowercased
+		 when setLowercase method called`, () => {
+			 const initialText = appModel.transformationText;
```

Comment:
Tests need to be explicit:
appModel.inputText = "HI THERE";
expect(appModel.inputText).toBe("HI THERE");
appModel.setLowerCase();
expect(appModel.inputText).toBe("hi there");

Reply:
Done